### PR TITLE
OADP 3183 Doc improvements - Installing and configuring OADP & Uninst…

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP 1.3.0

OCP 4.13+

Resolves - https://issues.redhat.com/browse/OADP-3183

Deploy preview - 

https://69508--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#verifying-oadp-installation-1-2_installing-oadp-aws

Verification for 1.2 & earlier

https://69508--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#verifying-oadp-installation-1-3_installing-oadp-aws

Verification for 1.3 

https://69508--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp#about-installing-oadp-on-multiple-namespaces_about-installing-oadp

Installation of OADP on multiple namespaces

https://69508--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/about-installing-oadp#cpu-and-memory-requirement-for-configurations

CPU memory

### CODE REVIEW

**Review by stillalearner — APPROVED** [review]
lgtm

File: modules/oadp-velero-cpu-memory-requirements.adoc [hunk 1]
```diff
@@ -63,11 +63,11 @@ Memory - Request 16Gi, Limits 40Gi
...
 
-3. Restic resource usage corresponds to the amount of data, and type of data. For example, many small files or large amounts of data can cause Restic to utilize large amounts of resources. The https://velero.io/docs/v1.11/customize-installation/#customize-resource-requests-and-limits/[Velero] documentation references 500m as a supplied default, for most of our testing we found 200m request suitable with 1000m limit.  As cited in the Velero documentation, exact CPU and memory usage is dependent on the scale of files and directories, in addition to environmental limitations.
+3. Restic resource usage corresponds to the amount of data, and type of data. For example, many small files or large amounts of data can cause Restic to utilize large amounts of resources. The https://velero.io/docs/v1.11/customize-installation/#customize-resource-requests-and-limits/[Velero] documentation references 500m as a supplied default, for most of our testing we found a 200m request suitable with 1000m limit.  As cited in the Velero documentation, exact CPU and memory usage is dependent on the scale of files and directories, in addition to environmental limitations.
```
Comment (tmalove) [inline modules/oadp-velero-cpu-memory-requirements.adoc:66]:
```suggestion
3. Restic resource usage corresponds to the amount of data, and type of data. For example, many small files or large amounts of data can cause Restic to use large amounts of resources. The https://velero.io/docs/v1.11/customize-installation/#customize-resource-requests-and-limits/[Velero] documentation references 500m as a supplied default, for most of our testing we found a 200m request suitable with 1000m limit.  As cited in the Velero documentation, exact CPU and memory usage is dependent on the scale of files and directories, in addition to environmental limitations.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u

**Review by tmalove — APPROVED** [review]

File: modules/oadp-velero-cpu-memory-requirements.adoc (hunk 1 above)
Comment (CarmiWisemon) [reply modules/oadp-velero-cpu-memory-requirements.adoc:66]: Done

**Pull request merged**